controller/scheduler/fleet.py

```diff
@@ -377,6 +376,7 @@ def attach(self, name):
 
 CONTAINER_TEMPLATE = [
     {"section": "Unit", "name": "Description", "value": "{name}"},
+    {"section": "Service", "name": "EnvironmentFile", "value": "/etc/environment"},
```

Comment:
@bacongobbler It's covered in the documentation in the PR-- which was previously missing due to a botched rebase.  I've fixed that and the documentation is back now.